streamer/src/nonblocking/quic.rs

```diff
@@ -856,6 +896,20 @@ async fn handle_connection(
                             sleep(throttle_duration).await;
                         }
                     }
+                    let max_concurrent_uni_streams =
```

Comment:
nit: `max_concurrent_uni_streams` is very overloaded here. Can we simplify the code? Maybe we if use `min()` instead of `UniStreamQosUtil::max_concurrent_uni_streams_per_throttling_interval()`, the code could be compressed and simplified.

Reply:
I renamed the variables a little to clarify